msteams-platform/sbs-gs-javascript.yml

```diff
@@ -259,9 +259,9 @@ items:
 
     ## Take a tour of the tab app source code
 
-    Teams Toolkit provides all components for building an app. After creating the project, you can view the project folders and files in the **Explorer** area of Visual Studio Code.
+    Teams Toolkit provides all components for building an app. After creating the project, you can view the project folders and files in the **Explorer** in the Visual Studio Code.
 
-    :::image type="content" source="~/assets/images/teams-toolkit-v2/first-tab/folder-structure-tab-app.png" alt-text="structure tab":::
+    :::image type="content" source="~/assets/images/teams-toolkit-v2/first-tab/folder-structure-tab-app.png" alt-text="Screen shot shows the structure tab.":::
```

Comment:
The image seems outdated. "~/assets/images/teams-toolkit-v2/first-tab/folder-structure-tab-app.png"

Reply:
The following directory structure also needs to be updated.

Reply:
```
    When you add the cloud functionality, Teams Toolkit adds the necessary folders to the project. The `api` folder holds the code to any Azure Functions you write.
```
Teams Toolkit v5 does not provide `add feature` funcionality anymore. I wonder this note could be removed.

Reply:
Updated as suggested